src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: $trancition-duration;
+  }
+
+  &__link:hover {
```

Comment:
![image](https://user-images.githubusercontent.com/122156399/216054241-6697c3d9-bc57-4c5f-b6c8-4541df4fead6.png)
checklist third item, fix it please:3